pkgs/development/interpreters/php/default.nix

```diff
@@ -61,6 +62,7 @@ let
   , xmlrpcSupport ? (config.php.xmlrpc or false) && (libxml2Support)
   , cgotoSupport ? config.php.cgoto or false
   , valgrindSupport ? (config.php.valgrind or true) && (versionAtLeast version "7.2")
+  , ipv6Support ? config.php.ipv6 or false
```

Comment:
we should default to enabling this

Reply:
default ipv6 enabled
`++ optional (!ipv6Support) "--disable-ipv6";`
Need rename to `ipv6DisableSupport` ?

Reply:
ooops, fixed.